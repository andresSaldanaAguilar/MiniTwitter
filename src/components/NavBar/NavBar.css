.navBar {
  padding: 10px;
  font-family: arial;
  text-align: left;
}
