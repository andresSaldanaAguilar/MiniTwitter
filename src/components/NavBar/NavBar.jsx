/* eslint-disable jsx-a11y/anchor-is-valid */
import React from 'react';

const NavBar = () => (
  <div>
    <p>
      <a href="#">Home</a>
    </p>
    <p>
      <a href="#">Profile</a>
    </p>
    <p>
      <a href="#">Logout</a>
    </p>
  </div>
);

export default NavBar;
