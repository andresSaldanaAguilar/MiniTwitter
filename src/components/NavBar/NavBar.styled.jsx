import styled from 'styled-components';

export const NavBarContainer = styled.div`
  width: 100%;
  position: sticky;
  top: 0px;
  align-self: flex-start;
`;

export const LogoContainer = styled.div`
  background-color: rgb(67, 154, 254);
  padding: 12px 20px;
  display: flex;
  justify-content: center;
`;
