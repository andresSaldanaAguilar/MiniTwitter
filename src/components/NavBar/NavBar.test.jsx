// Link.react.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from './NavBar';

test('Link changes the class when hovered', () => {
  const component = renderer.create(<NavBar />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
