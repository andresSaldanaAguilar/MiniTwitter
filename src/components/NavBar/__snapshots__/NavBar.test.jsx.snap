// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <p>
    <a
      href="#"
    >
      Home
    </a>
  </p>
  <p>
    <a
      href="#"
    >
      Profile
    </a>
  </p>
  <p>
    <a
      href="#"
    >
      Logout
    </a>
  </p>
</div>
`;
